axon-framework/spring-boot-integration.md

```diff
@@ -32,36 +32,78 @@ To register Axon's JPA Entities, include the relevant packages, as described bel
 ## Demystifying Axon Spring Boot Starter
```

Comment:
I think the `JPA & Persistence Contexts` section above would benefit from naming the difference between the Jakarta (default) and Javax (under `legacyjpa`) packaging adjustment in AF.